NTupleAnalyzerHH_nanoAOD.py

```diff
@@ -940,41 +1020,33 @@ def LeptonsAndJetsFromHH(T):
 		else:
 			matchedrecomuons.append(emptyvector)
 			recomuoninds.append(-99)
-		#print 'dr', mindr, 'pt', muons[bestrecomuonind].Pt(), g.Pt(), 'eta', muons[bestrecomuonind].Eta(), g.Eta(), 'phi', muons[bestrecomuonind].Phi(), g.Phi()
 	while len(recomuoninds)<2 :
 		recomuoninds.append(-99)
-	#return([genmuons,matchedrecomuons,recomuoninds])
-	#return(recomuoninds)
 
+###################### genMatching Electrons ##########################
 	for n in range(T.nElectron):
 		m = TLorentzVector()
-		m.SetPtEtaPhiM(T.Electron_pt[n],T.Electron_eta[n],T.Electron_phi[n],0)
+		m.SetPtEtaPhiM(T.Electron_pt[n],T.Electron_eta[n],T.Electron_phi[n],T.Electron_mass[n])
```

Comment:
This looks like a problem in the indentation compared to the old version, please check.